lib/pipekit/field_repository.rb

```diff
@@ -1,25 +1,34 @@
 module Pipekit
   module FieldRepository
-      def get_by_key(key)
-        key = Config.field_id(parent_resource, key)
-        search_fields("key", key)
-      end
+    include Repository
```

Comment:
This feels kinda like sneaky inheritance-by-composition. Maybe it's the naming? Just all the `initialize` stuff in the `Repository` module, I'm like...y u no inherit (obv 'prefer composition to inheritance' but _this_ hard and fast?)